lib/webauthn/authentication_service.rb

```diff
@@ -49,11 +53,12 @@ def authenticate_security_key
       # 13. Verify that the User Present bit of the flags in authData is set.
```

Comment:
Hmm have the steps here gone out of sync with  https://w3c.github.io/webauthn/#sctn-verifying-assertion? This should be step 17 and does this mean our implementation has gone out of sync?

Reply:
Yes, good catch, our implementation is out of sync. I will look into this shortly, there is a reordering change here (spec has a few more steps in the beginning) but also some steps have different changes included.

Reply:
Thanks :+1: I think the changes can be done in another PR to make it easier to review though.